libcxx/include/__chrono/tzdb.h

```diff
@@ -31,6 +37,9 @@ namespace chrono {
 
 struct _LIBCPP_AVAILABILITY_TZDB tzdb {
   string version;
+  vector<pair<string, vector<__tz::__rule>>> __rules;
```

Comment:
Also, can we make this a private member?

Reply:
We can't. I've removed the vector here and added it to the pimpl of `tzdb_list`.